tests/test_wps_split_merged_climos.py

```diff
@@ -3,19 +3,49 @@
 from pywps import Service
 from pywps.tests import assert_response_success
 
-from .common import client_for, TESTDATA
+from .common import client_for, TESTDATA, local_path, opendap_path
 from thunderbird.processes.wps_split_merged_climos import SplitMergedClimos
 
-gcm_360_climos_local = TESTDATA["test_local_gcm_360_climos_nc"]
-gcm_climos_local = TESTDATA["test_local_gcm_climos_nc"]
-tasmax_climos_local = TESTDATA["test_local_tasmax_climos_nc"]
-gcm_360_climos_opendap = TESTDATA["test_opendap_gcm_360_climos_nc"]
-gcm_climos_opendap = TESTDATA["test_opendap_gcm_climos_nc"]
-tasmax_climos_opendap = TESTDATA["test_opendap_tasmax_climos_nc"]
+# limiting test_data to climo files
+climo_local_files = [
+    local_path(nc) for nc in TESTDATA["test_local_nc"] if nc.endswith("_climos.nc")
+]
+climo_opendaps = [
+    opendap_path(opendap)
+    for opendap in TESTDATA["test_opendaps"]
+    if opendap.endswith("_climos.nc")
+]
 
 client = client_for(Service(processes=[SplitMergedClimos()]))
 
 
+def build_netcdf_sets(climo_files):
+    """
+    This function takes a list of climo_files and returns a list of
+    combinations of climo_files. The purpose of this it is to make
+    test cases for wps_split_merged_climos to handle multiple inputs.
+    For example if [climo1, climo2, climo3] is given as an input, it
+    returns the following output: 
+    [
+        (climo1, climo2),
+        (climo1, climo3),
+        (climo2, climo3),
+        (climo1, climo2, climo3) <--- This is the worst case scenarioW
+    ]
+    """
+    netcdf_sets = []
+
+    for first_idx, first_component in enumerate(climo_files):
+        second_idx = first_idx + 1
+        while second_idx < len(climo_files):
+            second_component = climo_files[second_idx]
+            netcdf_sets.append((first_component, second_component))
+            second_idx += 1
+
+    netcdf_sets.append(tuple(climo_files))
+    return netcdf_sets
```

Comment:
I still don't love this, but it is much better.  I think it is ok for now.  We can always update it in the future.

Reply:
Sorry, I actually did not read your reply to my comment about this until now. I have fixed it in the new commit.